src/test/java/org/mskcc/oncokb/controller/TrialsControllerTest.java

```diff
@@ -0,0 +1,141 @@
+package org.mskcc.oncokb.controller;
+
+import org.bson.Document;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mskcc.oncokb.MatchminerCurateApp;
+import org.mskcc.oncokb.model.Clinical;
+import org.mskcc.oncokb.model.Genomic;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.junit4.SpringRunner;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@Configuration
+@SpringBootTest(classes = MatchminerCurateApp.class)
+public class TrialsControllerTest {
+    private List<Clinical> clinicals;
+    private List<Genomic> genomics;
+    private JSONArray clinicalArray;
+    private JSONArray genomicArray;
+    private Set<Document> trialMatchDocs;
+    private TrialsController trialsController;
+
+    @Before
+    public void setup() {
+        clinicals = new ArrayList<>();
+        genomics = new ArrayList<>();
+        clinicalArray = new JSONArray();
+        genomicArray = new JSONArray();
+        trialMatchDocs = new HashSet<>();
+        trialsController = new TrialsController();
+
+        try {
+            clinicals.add(new Clinical("BRCA-METABRIC-S1-MB-0506","Parker Swinford [fake] M.D.",
+                "Parker@fake_email.com", "Breast Invasive Ductal Carcinoma",
+                "2015-06-15", "alive", "Shirley Kavanagh[Fake]", "1961-12-17",
+                0, "female"));
+            clinicals.add(new Clinical("BRCA-METABRIC-S1-MB-0877", "1937-04-30", "female"));
+            genomics.add(new Genomic("BRCA-METABRIC-S1-MB-0506", "PIK3CA",
+                "p.H1047R", "Missense_Mutation", "MUTATION", null,
+                false, "chr3", "178952085", "c.A900G", "A",
+                8, "-", 0.5827659956067299, 3));
+            genomics.add(new Genomic("BRCA-METABRIC-S1-MB-0506", "TP53",
+                "p.C135W", "Missense_Mutation", "Mutation", null,
+                false, "chr17", "7578525", "c.G531C", "G",
+                8, "-", 0.5721701032601486, 2));
+            genomics.add(new Genomic("BRCA-METABRIC-S1-MB-0877", "BRAF",
+                "T599_V600insEAT", "Silent", "MUTATION"));
+
+            clinicalArray.put(new JSONObject("{\"ORD_PHYSICIAN_EMAIL\":\"Parker@fake_email.com\"," +
```

Comment:
Could you lay out the framework to do multiple tests? I think there is another way to do instead of manually copy the JSON data here. Similar to Parameterized test. You can find examples in oncokb core module.